#request
geometry_msgs/Pose startPosition
---
#response
geometry_msgs/Pose RCM
